### ISSUE TITLE
Updated from a

### ISSUE THREAD

**Issue by animosity22** [pkkid/python-plexapi#622]
**Describe the issue**
I recently rebuilt and a machine and since updated versions of PlexAPI.

I used to use:

```
VERSION = '3.6.0'
```

and now the latest:

```
VERSION = '4.2.0'
```

**Code snipppets**
I use a pretty simple item to mark things watched based on label that's been running for quite a number of months:

```
#!/usr/bin/python3

import os
import subprocess
import sys
os.environ['PLEXAPI_CONFIG_PATH'] = "/home/felix/scripts/plexapi.ini"

from plexapi.server import PlexServer
plex = PlexServer()

tv = plex.library.section('TV Shows')
for shows in tv.search(label='AutoWatch'):
    for episode in shows.unwatched():
        episode.markWatched()
```

It generates an error as label doesn't seem to be there anymore:

```
Traceback (most recent call last):
  File "./plex-mark-watched", line 12, in <module>
    for shows in tv.search(label='AutoWatch'):
  File "/home/felix/.local/lib/python3.8/site-packages/plexapi/library.py", line 669, in search
    args[category] = self._cleanSearchFilter(category, value, libtype)
  File "/home/felix/.local/lib/python3.8/site-packages/plexapi/library.py", line 717, in _cleanSearchFilter
    raise BadRequest('Unknown filter category: %s' % category)
plexapi.exceptions.BadRequest: Unknown filter category: label
```

**Expected behavior**

**Enviroment (please complete the following information):**
Ubuntu 20.04
Python3 / other versions noted above.

Is there something in the later version that should be updated to point to instead?

**Comment by blacktwin** [comment]
Confirming issue. The proper filter name is `show.label` but that also will not return results as the the query is of `show.label` is incorrect. I think I can fix in #598 by adding a check for a libtype when a `.` separator exists and to drop the `libtype.`  but that may not always be the case.

**Comment by blacktwin** [comment]
This is a duplicate issue, though first and following the template, there is more discussion in #623 . Closing, please use #623 going forward.
